code/__DEFINES/cooldowns.dm

```diff
@@ -65,7 +64,6 @@
 //Shared cooldowns for actions
 #define MOB_SHARED_COOLDOWN_1 (1<<0)
 #define MOB_SHARED_COOLDOWN_2 (1<<1)
-#define MOB_SHARED_COOLDOWN_3 (1<<2)
```

Comment:
I would also keep this one so people don't get confused about how these cooldown groups work